src/openrct2/platform/Platform.Linux.cpp

```diff
@@ -13,8 +13,11 @@
 #    include <pwd.h>
 #    if defined(__FreeBSD__)
 #        include <stddef.h>
-#        include <sys/sysctl.h>
+// clang-format off
+// formatting check is disabled so that u_int, typedef'd in sys/types.h may be included before including sys/sysctl.h
 #        include <sys/types.h>
```

Comment:
`// clang-format off` can be avoided if you replace `<sys/types.h>` with `<sys/param.h>`. The latter already includes `<sys/types.h>` and is frequently used together with `<sys/sysctl.h>` due to `__FreeBSD_version` and `nitems()` e.g., try replacing `4` with `nitems(mib)` in
https://github.com/OpenRCT2/OpenRCT2/blob/2bffda8aff309ef0642190983b2fd2384553abb2/src/openrct2/platform/Platform.Linux.cpp#L155

`KERN_PROC_PATHNAME` is also supported on DragonFly and NetBSD. The latter actually [requires](http://netbsd.gw.com/cgi-bin/man-cgi?sysctl+3+NetBSD-current) `<sys/param.h>` rather than `<sys/types.h>`.

Reply:
@jbeich do you consider it done now? If not please leave a comment or submit a PR